### PR TITLE
On comment ticket, validation implementation. Response returned. Translations.

### PR DESCRIPTION
**Pull request by aleixgil**
To avoid comments changing html url request (ticket id). Checking ticket author and user requester.
To get more control about request params and response return.

Linear: https://linear.app/revo/issue/REV-4128/edicio-de-tickets-handesk

Reviewer: @PauBenetPrat

### CODE REVIEW

File: resources/lang/en/validation.php [hunk 1]
```diff
@@ -148,4 +148,8 @@
         'subject'               => 'subject',
         'message'               => 'message',
     ],
+    'emptyBodyComment'          => 'Can not create a comment with empty body.',
+    'commentCreated'            => 'Comment created.',
+    'ticketCommentInjection'    => 'Error submitting comment. It is not being sent to the correct ticket.',
+    'solvedTicket'              => 'ticket résolu.',
```
Comment (PauBenetPrat) [inline resources/lang/en/validation.php:154]: Ticket amb majuscula @aleixgil

File: resources/lang/ca/validation.php [hunk 2]
```diff
@@ -141,4 +141,8 @@
         'subject'               => 'assumpte',
         'message'               => 'missatge',
     ],
+    'emptyBodyComment'          => 'No pots crear un comentari buit.',
+    'commentCreated'            => 'Comentari creat.',
+    'ticketCommentInjection'    => 'Error al enviar el comentari. No s\'està guardant al ticket que toca.',
+    'solvedTicket'              => 'Ticket solucionat.',
```
Comment (PauBenetPrat) [inline resources/lang/ca/validation.php:147]: tiquet va amb qu no?

Comment (aleixgil) [reply resources/lang/ca/validation.php:147]: cert

File: resources/lang/en/validation.php (hunk 1 above)
Comment (aleixgil) [reply resources/lang/en/validation.php]: fet

File: resources/lang/ca/validation.php (hunk 2 above)
Comment (aleixgil) [reply resources/lang/ca/validation.php]: fet

**Review by BadChoice — APPROVED** [review]

**Comment by aleixgil** [comment]
@BadChoice puja release pls!

**Pull request merged**